### ISSUE TITLE
What's the difference between 'log_config' and 'evaluation' in configuration files?

### ISSUE THREAD

**Issue by Hggang** [open-mmlab/mmaction2#1338]
In my opinion, when the 'log_config = dict(interval=20)' and 'evaluation = dict(interval = 5)', it means we will evaluate the train process every 5 iterations and store the results every 20 iterations.  This process is puzzling. Why they are not consistent parameters? such as evaluating the training process and saving the results in both 5 iterations. Maybe 'evaluation' goal is to save the model parameters instead of the results, and finally save the best model parameters every epoch?

**Comment by kennymckormick** [comment]
Hi, Hggang, 'log_config = dict(interval=20)' means print log every 20 **iterations**,  'evaluation = dict(interval = 5)' means evaluate the ckpt on th validation set every 5 **epochs** (only if the --validate flag is on).
